x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_actions/clone_layer_action.tsx

```diff
@@ -22,11 +22,11 @@ export const getCloneLayerAction = (props: CloneLayerAction): LayerAction => {
   });
 
   return {
-    id: 'cloneLayerAction',
     execute: props.execute,
     displayName,
     isCompatible: Boolean(props.activeVisualization.cloneLayer && !props.isTextBasedLanguage),
     icon: 'copy',
     'data-test-subj': `lnsLayerClone--${props.layerIndex}`,
+    order: -Infinity + 1,
```

Comment:
What's this?
If it's to put it above the other `-Infinity` I think it's not going to work...
`- Infinity + 1 === - Infinity // true`

Reply:
Good catch. The order is correct... but it must be by accident. For some reason I was thinking that `Infinity` in JS was just an alias for the largest float.